hack/migrate-to-antrea.sh

```diff
@@ -0,0 +1,247 @@
+        --from <CNI>                The CNI to be migrated from. Only support 'calico', 'flannel' now.
+        --networkpolicy             Convert NetworkPolicy to Antrea NetworkPolicy or not.
+        --dry-run                   Print the commands that would be executed, but do not execute them.
+        --backup                    Back up the original CNI configuration files and network policies.
+        --output-dir <DIR>          Output directory for the backup files.
+        --kubeconfig <KUBECONFIG>   Path to kubeconfig file.
+        --calico-manifest <PATH>    Path to the Calico manifest file, if not specified, the latest manifest will be used.
+        --flannel-manifest <PATH>   Path to the Flannel manifest file, if not specified, the latest manifest will be used.
+        --antrea-manifest <PATH>    Path to the Antrea manifest file, if not specified, the latest manifest will be used.
+"
+
+_calico_warning_message="
+WARNING: You are trying to migrate from Calico to Antrea. Please NOTE:
+If you are using Kubernetes origin Networkpolicy, all the network policies will still work
+after migration.
+During the migration, the workloads will be restarted and the network will be interrupted.
+If you are using Calico NetworkPolicy or Calico GlobalNetworkPolicy, only PART of
+the network policies could be converted to Antrea NetworkPolicy and Antrea ClusterNetworkPolicy.
+Please check the migration document for details."
+
+_flannel_warning_message="
+WARNING: You are trying to migrate from Flannel to Antrea. Please NOTE:
+During the migration, the workloads will be restarted and the network will be interrupted.
+"
+
+function print_usage {
+    echo "$_usage"
+}
+
+FROM=""
+NETWORK_POLICY=false
+DRY_RUN=false
+BACKUP=false
+OUTPUT_DIR=$(pwd)
+KUBECONFIG="$HOME/.kube/config"
+PLATFORM=$(uname | tr '[:upper:]' '[:lower:]')
+CALICO_MANIFEST="https://docs.projectcalico.org/manifests/calico.yaml"
+FLANNEL_MANIFEST="https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml"
+ANTREA_MANIFEST="https://raw.githubusercontent.com/antrea-io/antrea/master/build/yamls/antrea.yml"
+FROM_MANIFEST=""
+
+while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case $key in
+    --from)
+        FROM="$2"
+        shift 2
+        ;;
+    --networkpolicy)
+        NETWORK_POLICY=true
+        shift
+        ;;
+    --dry-run)
+        DRY_RUN=true
+        shift
+        ;;
+    --backup)
+        BACKUP=true
+        shift
+        ;;
+    --output-dir)
+        OUTPUT_DIR="$2"
+        if [ $OUTPUT_DIR == "" ]; then
+            echo "Warning: output directory is not specified, the backup files will be saved in the current directory"
+            OUTPUT_DIR=$(pwd)
+        fi
+        shift 2
+        ;;
+    --kubeconfig)
+        KUBECONFIG="$2"
+        shift 2
+        ;;
+    --from-manifest)
+        FROM_MANIFEST="$2"
+        shift 2
+        ;;
```

Comment:
This can be removed, and --calico-manifest, --flannel-manifest are still missing.